TracksDemo/TracksDemo.xcodeproj/project.pbxproj

```diff
@@ -792,7 +858,7 @@
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				INFOPLIST_FILE = "TracksDemo Mac/Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks";
-				MACOSX_DEPLOYMENT_TARGET = 10.14;
+				MACOSX_DEPLOYMENT_TARGET = 11.0;
```

Comment:
Similar comment as the iOS 13.0 vs 14.0 above. Should we set this as 10.13 and do the same on the `Podfile`?

Reply:
Addressed in 9ed5b7a08e4ec26b97358489e1741d2cd5451c7c, Podfile in 58f90120b5e589774357aacb8fe353e48aa72663